python/cugraph/cugraph/tests/community/test_balanced_cut.py

```diff
@@ -98,3 +98,17 @@ def test_edge_cut_clustering_directed_graph(partitions):
     )
     with pytest.raises(ValueError):
         cugraph_call(G_edge, partitions)
+
+
+@pytest.mark.sg
+def test_spectral_balanced_cut_clustering_deprecation_warning():
+    G = karate.get_graph(
+        create_using=cugraph.Graph(directed=False), ignore_weights=True
+    )
+    warning_msg = (
+        "spectralBalancedCutClustering is deprecated and will be removed in a future "
+        "release. Use spectralModularityMaximizationClustering instead."
+    )
+
+    with pytest.warns(DeprecationWarning, match=warning_msg):
```

Comment:
We always use `FutureWarning` for deprecations, because `DeprecationWarning` is hidden from the user by default.
```suggestion
    with pytest.warns(FutureWarning, match=warning_msg):
```